guacamole/src/main/webapp/app/form/templates/redirectField.html

```diff
@@ -0,0 +1,8 @@
+<div class="redirect-field-container">
+    <div class="redirect-field">
+        <p ng-show="redirectMessage"
+           translate="{{redirectMessage.key}}"
+           translate-values="{{redirectMessage.variables}}">
```

Comment:
1. Does `redirectMessage` here need to be `field.redirectMessage`?
2. The getter on the Java side for this is `getTranslatableMessage()`, which would be serialized within the JSON as `translatableMessage` unless overridden by an annotation. I think this will need to either be referenced as-is (`translatableMessage`) or annotated such that it's exposed as `redirectMessage`.